.github/workflows/dev-deploy-api.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       contents: read
       packages: write
       deployments: write
+      id-token: write
```

Comment:
This permission enables access to [GitHub's OIDC token endpoint](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect). This means GitHub will generate a short-lived OIDC token for each build, and Depot uses that token to authenticate. This is in place of a static access token, so you don't have to create one or manage key rotation ✨